pkg/project/formatter.go

```diff
@@ -30,7 +33,7 @@ func Format(files []string, basePath string, flatten bool, formatJSON bool, writ
 		identityResolver := identity.NewResolver(identity.DefaultLifecycleIdentity)
 
 		if flatten {
-			notebook, err := editor.Deserialize(data, identityResolver)
+			notebook, err := editor.Deserialize(logger, data, identityResolver)
```

Comment:
Remark: With my earlier suggestion, this will become:
```suggestion
			notebook, err := editor.Deserialize(data, editor.Options{IdentityResolver: identityResolver})
```

The logger can be figure out by `editor.Deserialize` and default to noop.